src/components/AdminPanelLayout/AdminPanelLayout.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {Layout} from '../../pages/Layout/Layout';
+import {Col, ListGroup, Row, TabContainer, TabContent} from 'react-bootstrap';
+import {StatisticsPane} from './StatisticsPane';
+import {ClusterPane} from './ClusterPane';
+import UsersPane from './UsersPane';
+import {DowntimeScheduler} from './DowntimeScheduler';
+
+interface AdminPanelLayoutProps {
+  defaultActivePanel?: '#statistics'|'#cluster-management'|'#user-management';
+}
+
+export const AdminPanelLayout = (props: AdminPanelLayoutProps) => (
```

Comment:
I recommend making an array of objects that contains the label, the key, and the component to render so that you don't have to duplicate the ListGroup.Item and TabPane statements in the other component files. I would feel better if the keys of the tab pane stay in one file to make it more clear.

ex.
```
const panes = [
{label:  ' Application Statistics', eventKey: '#statistics', component: StatisticsPane}
...
];
```
render the labels:
```
{panes.map(pane => (
  <ListGroup.Item key={pane.eventKey} action={true} href={pane.eventKey}>
	{pane.label}
  </ListGroup.Item>
))}
```

and render the panes themselves: (this assumes the tab pane has been extracted from each component.

```
<TabContent>
	{panes.map(pane => (
	  <TabPane key={pane.eventKey} eventKey={pane.eventKey}>
		{pane.component}
	  </TabPane>
	))}
</TabContent>
```

Reply:
Hey Jason, I pushed these changes, but for some reason the components within the `<TabContent>` aren't loading. I got this error in the console, but I can't work out how to fix it.

![image](https://user-images.githubusercontent.com/15639276/123721640-bde9ca00-d854-11eb-8215-9b49ace747c7.png)

Any ideas?

Reply:
Updated this, ready for you to take another look.